sdk/tests/graph/test_analyzers.py

```diff
@@ -0,0 +1,91 @@
+#
```

Comment:
I think this a big good to have for now. I guess I am influenced by golang style tests where they iterate through test cases instead of writing new test methods.